client/lib/phone-validation/test/index.jsx

```diff
@@ -1,71 +1,34 @@
-			'phone_number_contains_special_characters'
-		);
-	} );
-	test( 'should fail an invalid number', () => {
-		assert.strictEqual( phoneValidation( '+111111111' ).error, 'phone_number_invalid' );
-	} );
-	test( 'should pass a valid number', () => {
-		assert.strictEqual( phoneValidation( '+447941952721' ).info, 'phone_number_valid' );
-	} );
-	test( 'should pass a valid number in US 463 area code', () => {
-		assert.strictEqual( phoneValidation( '+14631234567' ).info, 'phone_number_valid' );
-	} );
-	test( 'should pass a valid 10 digit argentine without leading 9 number', () => {
-		assert.strictEqual( phoneValidation( '+542231234567' ).info, 'phone_number_valid' );
-	} );
-	test( 'should pass a valid 10 digit argentine with leading 9 number', () => {
-		assert.strictEqual( phoneValidation( '+5492231234567' ).info, 'phone_number_valid' );
-	} );
-	test( 'should pass a valid 8-digit croatian number', () => {
-		assert.strictEqual( phoneValidation( '+38598123456' ).info, 'phone_number_valid' );
-	} );
-	test( 'should pass a valid 8-digit danish number', () => {
-		assert.strictEqual( phoneValidation( '+4528123456' ).info, 'phone_number_valid' );
-	} );
-	test( 'should pass a valid 7-digit jamaican number', () => {
-		assert.strictEqual( phoneValidation( '+18761234567' ).info, 'phone_number_valid' );
-	} );
-	test( 'should pass a valid new format vietnamese number starting with 7', () => {
-		assert.strictEqual( phoneValidation( '+84761234567' ).info, 'phone_number_valid' );
-	} );
-	test( 'should pass a valid new format vietnamese number starting with 3', () => {
-		assert.strictEqual( phoneValidation( '+84361234567' ).info, 'phone_number_valid' );
-	} );
-	test( 'should pass a valid new format vietnamese number with leading zero', () => {
-		assert.strictEqual( phoneValidation( '+840361234567' ).info, 'phone_number_valid' );
-	} );
-	test( 'should pass a valid greenland number starting with 2', () => {
-		assert.strictEqual( phoneValidation( '+299239349' ).info, 'phone_number_valid' );
-	} );
-	test( 'should pass a valid qatari number', () => {
-		assert.strictEqual( phoneValidation( '+97466641333' ).info, 'phone_number_valid' );
-	} );
-	test( 'should pass a valid myanmar number with 7 digits after leading 9', () => {
-		assert.strictEqual( phoneValidation( '+9595512345' ).info, 'phone_number_valid' );
-	} );
-	test( 'should pass a valid myanmar number with 8 digits after leading 9', () => {
-		assert.strictEqual( phoneValidation( '+95942612345' ).info, 'phone_number_valid' );
-	} );
-	test( 'should pass a valid myanmar number with 9 digits after leading 9', () => {
-		assert.strictEqual( phoneValidation( '+959426123456' ).info, 'phone_number_valid' );
-	} );
-	test( 'should pass a valid Brazilian number starting with 119', () => {
-		assert.strictEqual( phoneValidation( '+5511961231234' ).info, 'phone_number_valid' );
+
+	test.each( [
+		{ input: '+447941952721', expected: 'phone_number_valid' }, // Valid United Kingdom number
```

Comment:
(Completely optional) -- I really like the cleanliness of the `test.each` here, but I wonder if we might be losing a tiny bit of utility in losing the semantic description of the case in the test name. Maybe we could add the comment description as its own property and surface it in the name?

Again, that may be overkill, just thought I'd throw out the idea!

Reply:
I debated about this, but I personally think that exposing the country code would be useful, the entire description of the test step as it stood prior to the refactor was unnecessary.

I'll take a look to see if I can make the test names a bit clearer.

Reply:
Changes implemented in c05881695f059a3028383c92e227b67e95a0d37a.

In the above changeset, I've made the additional change to remove the `expected` value from the parametrization as it was the same across the board for all numbers being validated.